qemu/tests/mlock_basic.py

```diff
@@ -7,6 +7,76 @@
 from virttest import error_context
 
 
+class MlockBasic(object):
+    """
+    Base class for mlock test
+    """
+    def __init__(self, test, params, env):
+        self.test = test
+        self.params = params
+        self.env = env
+        self.mlock_cmd = params["mlock_cmd"]
+        self.unevictable_cmd = params["unevictable_cmd"]
+        self.realtime_mlock = params["realtime_mlock"]
+        self.vm_mem = int(params["mem"])
+        self.vm = None
+        self.mlock_pre = None
+        self.mlock_post = None
+        self.unevictable_pre = None
+        self.unevictable_post = None
+
+    def get_mlock_unevictable(self):
```

Comment:
we can use `read_from_vmstat()` from utils_memory, for nr_mlock and nr_unevictable.

Reply:
Yes, you're right! Modified, Thanks!